odoo/http.py

```diff
@@ -999,6 +1025,7 @@ def delete_from_identifiers(self, identifiers):
         for fn in files_to_unlink:
             with contextlib.suppress(OSError):
                 os.unlink(fn)
+        return len(files_to_unlink)
```

Comment:
I did not follow for what is this used for within this revision ? If you remove, remove as well the docstring as the method is untouched.

Reply:
I added this return because it is a useful information when using this method (in an experimental context).
I have removed it.